vendor/bat-native-ledger/src/bat/ledger/internal/publisher/server_publisher_fetcher.cc

```diff
@@ -55,45 +53,38 @@ void ServerPublisherFetcher::Fetch(
   FetchCallbackVector& callbacks = callback_map_[publisher_key];
   callbacks.push_back(callback);
   if (callbacks.size() > 1) {
-    BLOG(1, "Fetch already in progress for publisher " << publisher_key);
+    BLOG(1, "Fetch already in progress");
     return;
   }
 
-  BLOG(1, "Fetching server publisher info for " << publisher_key);
+  const std::string hex_prefix =
+      GetHashPrefixInHex(publisher_key, kQueryPrefixBytes);
 
-  std::string hex_prefix = GetHashPrefixInHex(
+  auto url_callback = std::bind(&ServerPublisherFetcher::OnFetchCompleted,
+          this,
+          _1,
+          _2,
+          publisher_key);
+
+  private_cdn_server_->get_publisher()->Request(
       publisher_key,
-      kQueryPrefixBytes);
-
-  // Due to privacy concerns, the request length must be consistent
```

Comment:
This comment should be retained somewhere.

Reply:
added to `get_publisher`